modules/imgproc/src/colormap.cpp

```diff
@@ -499,11 +498,8 @@ namespace colormap
             CV_Error(Error::StsAssert, "cv::LUT only supports tables of size 256.");
         Mat src = _src.getMat();
         // Return original matrix if wrong type is given (is fail loud better here?)
```

Comment:
@mshabunin,   Partially same, yes.  My bad.  It was my 1st pull request on github. Should I cancel PR 6167?  And add commits from 6167 here?